src/components/MultiSelect/index.jsx

```diff
@@ -16,7 +16,14 @@ class MultiSelect extends Component {
     }
   }
 
+  componentWillUnmount() {
+    this.unmounted = true
```

Comment:
If you remember why we do a setState in a setTimeout, can you add a comment explaining it? I'm 100% sure future me will be like « meh, why? » in front of it :sweat_smile:

Reply:
> If you remember why we do a setState in a setTimeout

I don't :/